ios/Podfile

```
@@ -0,0 +1,44 @@
+# Uncomment this line to define a global platform for your project
+# platform :ios, '12.0'
```

:thinking: I think this makes sense?
```suggestion
platform :ios, '12.0'
```